src/docs/goose-book/src/getting-started/running.md

```diff
@@ -4,87 +4,42 @@ We will use Cargo to run our example load test application. It's best to get in
- GET /                   | 20.68      | 7          | 254        | 19
--------------------------------------------------------------------------------
- Slowest page load within specified percentile of requests (in ms):
- ------------------------------------------------------------------------------
- Name                    | 50%    | 75%    | 98%    | 99%    | 99.9%  | 99.99%
- -----------------------------------------------------------------------------
- GET /                   | 19     | 21     | 53     | 69     | 250    | 250
+$ cargo run --release -- --host https://drupal-memcache.ddev.site --report-file=report.html
+    Finished release [optimized] target(s) in 0.17s
+     Running `target/release/goose_attack --host 'https://drupal-memcache.ddev.site' --report-file=report.html`
+
+// Some output omitted...
 ```
 
-By default, Goose will hatch 1 `GooseUser` per second, up to the number of CPU cores available on the server used for load testing. In the above example, the server has 8 CPU cores, so it took 8 seconds to hatch all users. After all users are hatched, Goose flushes all metrics collected during the hatching process so all subsequent metrics are taken with all users running. Before flushing the metrics, they are displayed to the console so the data is not lost.
+The HTML report will include a few sections. Let's go through them: 
+
+![HTML report header section](report_header.png)
+
+By default, Goose will hatch 1 `GooseUser` per second, up to the number of CPU cores available on the server used for load testing. In the above example, the server has 8 CPU cores, so it took 8 seconds to hatch all users. After all users are hatched, Goose flushes all metrics collected during the hatching process (use `--no-reset-metrics` to disable this behavior) so all subsequent metrics are taken with all users running. Before flushing the metrics, they are displayed to the console so the data is not lost.
 
 The same metrics are displayed per-task and per-request. In our simple example, our single task only makes one request, so in this case both metrics show the same results.
 
-The per-task metrics are displayed first, starting with the name of our Task Set, `LoadtestTasks`. Individual tasks in the Task Set are then listed in the order they are defined in our load test. We did not name our task, so it simply shows up as "1: ". All defined tasks will be listed here, even if they did not run, so this can be useful to confirm everything in your load test is running as expected.
+![HTML report request metrics section](report_request_metrics.png)
+
+The per-request metrics are displayed first. Our single task makes a `GET` request for the `/` path, so it shows up in the metrics as `GET /`. The table in this section displays shows the total number of requests made (2,499),the average number of requests per second (208.25), and the average number of failed requests per second (0). 
```

Comment:
`number of requests made (2,499),the average number` there's a space missing after the comma.

Maybe add a little more information: "If our load test made multiple requests, the Aggregated line at the bottom of this section would show totals and averages of all requests together. Because we only make a single request, this row is identical to the per-request metrics."

Reply:
(Whoops, this would go under the next paragraph.)

Reply:
Done.